ui/plugins/controller.js

```diff
@@ -22,6 +22,8 @@ treeherder.controller('PluginCtrl', [
         $scope.job = {};
         $scope.artifacts = {};
 
+        var reftestUrlRoot = "http://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml#logurl=";
```

Comment:
Oh though the old one in pluginpanel.html isn't HTTPS. Maybe see if it still works with non-HTTPS log urls.